napari/layers/labels/labels.py

```diff
@@ -640,6 +643,7 @@ def selected_label(self, selected_label):
         if selected_label == self.selected_label:
             return
 
+        self.prev_selected_label = self.selected_label
```

Comment:
Won't this result in a loop of background to background once you select the background once?

Reply:
No, it should not result in a loop because of the if above. If you try to set the same color, the setter filters out that without changing anything.

Reply:
So, actually this if guarantees that `self.prev_selected_label` never equals to `self.selected_label`.